app/api/intendedGraduations/delete/func.js

```diff
@@ -0,0 +1,20 @@
+const app = require('@/app')
+const errors = require('@/errors')
+
+module.exports = async function(context){
+  let { graduationId } = context.params;
+  let intendedGraduation = await app.models.intendedGraduation.findOne({
+     graduation: graduationId,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 5.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
    graduation: graduationId,
```